spec/import_spec.rb

```diff
@@ -158,9 +162,11 @@ def upload_file!(name, ext='csv')
 
       context "without headers" do
         context "with known csv headers" do
-          let(:options) {
-            {template_object: ActiveAdminImport::Model.new(csv_headers: ['Name', 'Last name', 'Birthday'])}
-          }
+          
+          let(:options) do
+            attributes = { csv_headers: ['Name', 'Last name', 'Birthday'] }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.